src/store/getters/state-getters.ts

```diff
@@ -252,7 +257,9 @@ export const isBusySaving = (state: StateIF): boolean => {
 
 /** Is true when the step is valid. */
 export const isDefineCompanyValid = (state: StateIF): boolean => {
-  return getDefineCompanyStep(state).valid
+  const validCoopType = isTypeCoop(state) ? !!getCooperativeType(state) : true
```

Comment:
Please add a comment for this.

Reply:
(I had to think about this to understand it.)

Reply:
Definitely, will do. Will try to make it more clear.